test/e2e_env/multizone/localityawarelb/locality_aware_multizone_hybrid.go

```diff
@@ -0,0 +1,254 @@
+
+	It("should route based on defined strategy", func() {
+		Eventually(func() (map[string]int, error) {
+			return client.CollectResponsesByInstance(multizone.UniZone1, "demo-client_locality-aware-lb_svc", "test-server_locality-aware-lb_svc_80.mesh", client.WithNumberOfRequests(100))
+		}, "30s", "500ms").Should(
+			And(
+				HaveKeyWithValue(Equal(`test-server-node-1-zone-4`), BeNumerically("~", 93, 5)),
+				HaveKeyWithValue(Equal(`test-server-az-1-zone-4`), BeNumerically("~", 7, 5)),
+				Or(
+					HaveKeyWithValue(Equal(`test-server-node-2-zone-4`), BeNumerically("~", 2, 2)),
+					HaveKeyWithValue(Equal(`test-server-no-tags-zone-4`), BeNumerically("~", 2, 2)),
+					Not(HaveKeyWithValue(Equal(`test-server-node-2-zone-4`), BeNumerically("~", 2, 2))),
+					Not(HaveKeyWithValue(Equal(`test-server-no-tags-zone-4`), BeNumerically("~", 2, 2))),
+				),
+			),
+		)
+
+		// when app with the highest weight is down
+		Expect(multizone.UniZone1.DeleteApp("test-server-node-1")).To(Succeed())
+
+		// then traffic goes to the next highest
+		Eventually(func() (map[string]int, error) {
+			return client.CollectResponsesByInstance(multizone.UniZone1, "demo-client_locality-aware-lb_svc", "test-server_locality-aware-lb_svc_80.mesh", client.WithNumberOfRequests(100))
+		}, "30s", "500ms").Should(
+			And(
+				HaveKeyWithValue(Equal(`test-server-az-1-zone-4`), BeNumerically("~", 96, 5)),
+				Or(
+					HaveKeyWithValue(Equal(`test-server-node-2-zone-4`), BeNumerically("~", 2, 2)),
+					HaveKeyWithValue(Equal(`test-server-no-tags-zone-4`), BeNumerically("~", 2, 2)),
+				),
+			),
+		)
+
+		// when the next app with the highest weight is down
+		Expect(multizone.UniZone1.DeleteApp("test-server-az-1")).To(Succeed())
+
+		// then traffic goes to the next highest
+		Eventually(func() (map[string]int, error) {
+			return client.CollectResponsesByInstance(multizone.UniZone1, "demo-client_locality-aware-lb_svc", "test-server_locality-aware-lb_svc_80.mesh", client.WithNumberOfRequests(100))
+		}, "30s", "500ms").Should(
+			And(
+				HaveKeyWithValue(Equal(`test-server-node-2-zone-4`), BeNumerically("~", 50, 5)),
+				HaveKeyWithValue(Equal(`test-server-no-tags-zone-4`), BeNumerically("~", 50, 5)),
+			),
+		)
+
+		// when all apps in local zone down
+		Expect(multizone.UniZone1.DeleteApp("test-server-node-2")).To(Succeed())
+		Expect(multizone.UniZone1.DeleteApp("test-server-no-tags")).To(Succeed())
+
+		// then traffic goes to the zone with the next priority, kuma-5
+		Eventually(func() (map[string]int, error) {
+			return client.CollectResponsesByInstance(multizone.UniZone1, "demo-client_locality-aware-lb_svc", "test-server_locality-aware-lb_svc_80.mesh", client.WithNumberOfRequests(100))
+		}, "30s", "500ms").Should(
+			And(
+				HaveKeyWithValue(Equal(`test-server-zone-5`), BeNumerically("~", 100, 5)),
+			),
+		)
+
+		// when zone kuma-5 is disabled
+		Expect(YamlUniversal(`
+name: kuma-5
+type: Zone
+enabled: false
+`)(multizone.Global)).To(Succeed())
```

Comment:
I don't think you can do this. This can affect other tests that are run in parallel